lib/rubocop/cop/style/endless_method.rb

```diff
@@ -5,8 +5,11 @@ module Cop
     module Style
       # Checks for endless methods.
       #
-      # It can enforce either the use of endless methods definitions
-      # for single-lined method bodies, or disallow endless methods.
+      # It can enforce either the use of endless methods definitions for single-lined method bodies,
+      # or disallow endless methods.
```

Comment:
It's difficult to describe all the different styles now in a single compact paragraph, since it's a mixture of enforcing and allowing, and sometimes for single lines and sometimes not. Here's my suggestion:

It can enforce endless method definitions whenever possible. It can also disallow multiline endless method definitions or all endless definitions.